CSharpSeleniumWebdriverSample/README.md

```diff
@@ -0,0 +1,63 @@
+# CSharpSeleniumWebdriverSample
+
+This sample demonstrates how you might set up a CI build for a simple, static html page to perform end to end accessibility tests in a browser.
+
+## Getting Started
+
+The individual files in the sample contain comments that explain the important parts of each file in context.
+
+Some good places to start reading are:
+
+* [tests/SampleTest.cs](./tests/SampleTest.cs): C# test file that opens [src/samplePage.html](./src/samplePage.html) in a browser with Selenium and runs accessibility scans against it with Selenium.WebDriver.ChromeDriver and Selenium.WebDriver.GeckoDriver
+* [azure-pipelines.yml](./azure-pipelines.yml): Azure Pipelines config file that sets up our Continuous Integration and Pull Request builds
+
+## Tools and libraries used
+
+* [Microsoft.NET.Test.Sdk](https://github.com/microsoft/vstest/) to enable running tests, collect diagnostics data and report results.
```

Comment:
We can probably slim down this list a bit; I think we can combine the first two bullets into just ".NET Core" and the third bullet into just "MSTest"

Reply:
They first 2 bullets have different project URL, the third one is supposed to be 2 packages with the same project URL, that's why I merged them in one bullet

Reply:
I think for the purposes of the readme, it's more helpful to use project names as they would be recognized by a typical user, not necessarily just enumerating all the NuGet package names we happen to depend on. In particular, I think saying that we use ".NET Core" and "MSTest" adequately imply all of the other dependencies in the first 3 bullet points, but would be more concise and more likely to be recognized offhand by a consumer of the sample